scripts/proto_codegen.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Regenerate python code from OTLP protos in
+# https://github.com/open-telemetry/opentelemetry-proto
+#
+# Optional envars:
+#   PROTO_REPO_DIR - the path to an existing checkout of the opentelemetry-proto repo
+#   PROTO_REPO_BRANCH - the branch or commit to build from
+
+set -e
+
+if [ -z "$VIRTUAL_ENV" ]; then
+    echo '$VIRTUAL_ENV is not set, you probably forgot to source it. Exiting...'
```

Comment:
Do you mean add a tox environment so running this script would just be `tox -e proto_codegen`?

Reply:
There is already `py3{4,5,6,7,8}-test-proto` which tests that `opentelemetry-proto` can be imported. It would be better to add this script as a pre-command of this tox directive.

Reply:
I was thinking this script would be run manually when there are upstream changes in OT-proto, because the changes could be breaking and require opentelemetry-proto to be updated. For instance, when I ran it today, it introduced breaking changes from a new PR that was merged.

Reply:
Ok, I see. In that case this should be good to go :+1:

Reply:
Awesome! @toumorokoshi can I marked this resolved or should I handle this another way?